tests/iface_namingmode/test_iface_namingmode.py

```diff
@@ -755,7 +755,7 @@ def test_show_arp(self, duthosts, enum_rand_one_per_hwsku_frontend_hostname, set
 
         for item in arptable['v4']:
             # To ignore Midplane interface, added check on what is being set in setup fixture
-            if (arptable['v4'][item]['interface'] != 'eth0') and (arptable['v4'][item]['interface'] not in minigraph_portchannels) and (item in setup['port_name_map']):
```

Comment:
I believe the intention here is to skip management port, which sonic always hard code it as 'eth0'

Is the test failing because eth0 was also used as alias?

If 'eth0' is both alias and mgmt port, wouldn't the test fail in different way?

Reply:
switch may also have some USB-Ethernet adapters so test will fail at `setup['port_name_map'][detail['interface']])` with KeyError because setup['port_name_map'] contains only aliases for switch-native ports read from file port_config.ini.
So I've decided to check if a given interface has an alias configured before doing assertions

Reply:
Can you give the command outputs with such USB device attached to help understand the problem?

Reply:
sure so here is a capture of command output
![image](https://user-images.githubusercontent.com/79265382/115224395-78eb3e00-a115-11eb-8918-6506163ba643.png)

Reply:
@yxieca please take a look at the screenshot above to make sure the fix makes sense

Reply:
Thanks, @antonpatenko from your command output, I don't see a reason to remove 'eth0', as the interface name is  usb0. I still don't understand why you removed the check of arptable['v4'][item]['interface'] != 'eth0'.

Reply:
the test sets aliases for interfaces found in port_config.ini and keeps such interfaces list in `setup['port_name_map']`.
By checking aliases for only the interfaces containing in `setup['port_name_map']` I'm skipping any custom setup interfaces like usb0, eth0 etc so there is no need to list each one explicitly

Reply:
@yxieca please take a look, thank you in advance

Reply:
The change looks good to me. Originally condition '(item in setup['port_name_map'])' is incorrect. `item` is IP address. Keys of dict `setup['port_name_map']` is interface name. This condition will always be false.
With the fix, condition `(arptable['v4'][item]['interface'] != 'eth0')` would be unnecessary.